scripts/scil_smooth_streamlines.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+    This script will smooth the streamlines, usually to remove the
```

Comment:
I would remove the indentation.

Also, a new sentence here:
`wiggles' in probabilistic tracking. Two choices of methods are available:`

And remove the additional indent for the warning paragraph.